docs/Reference/init.md

```diff
@@ -1,14 +1,16 @@
 {% include-markdown './init.gen.md' %}
+
 ## How It Works
 
-The `quix init` command sets up the initial Quix files based on your existing repository. When executed, it generates the necessary configuration files such as `dockerfile` and `app.yaml` for your application. 
+The `quix init` command sets up the initial Quix files based on your existing repository. When executed, it generates essential configuration files like `dockerfile`, `app.yaml`, and `.env` for your application, ensuring that your development environment is properly configured for seamless development and deployment.
+
+If run at the root level of your project, it also creates a `quix.yaml` file if one does not exist, and attempts to generate any missing `app.yaml`, `dockerfile`, and `.env` files for all applications within the project. The command intelligently customizes these files based on the detected programming language, dependencies, and project structure, making it easier to containerize and manage your applications.
```

Comment:
`root folder` instead of `root level` is better IMO to avoid any possible confusion between root access level and project root

Reply:
There are some other places too, but looks like based on sentence can't just bulk find and replace either.